app/crud.py

```
@@ -0,0 +1,63 @@
+        raise HTTPException(
+            status_code=500, detail=f"Error retrieving ToDo with ID:{todo_id}: {e}"
+        ) from e
+
+
+async def update_todo(db: AsyncSession, todo_id: int, todo: ToDo) -> ToDo:
+    try:
+        db_todo = await get_todo(db, todo_id)
+        for key, value in todo.dict(exclude_unset=True).items():
+            setattr(db_todo, key, value)
+        await db.commit()
+        return db_todo
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)